Jenkinsfile

```diff
@@ -0,0 +1,4 @@
+// https://github.com/Rudd-O/shared-jenkins-libraries
+@Library('shared-jenkins-libraries@master') _
```

Comment:
what do these do? Does having jenkins give us any advantage?

Reply:
Actually, I don't have to add the `Jenkinsfile`.  I'll remove this from the commit.